### DISCUSSION TITLE
Usage with Webpack wrapper (Webpack Encore)

### DISCUSSION THREAD

**Discussion by YummYume** [web-infra-dev/rspack#4438] [Q&A]
Hello!

Tired of slow builds, I am wondering how this could be used with a Webpack wrapper such as [Webpack Encore](https://github.com/symfony/webpack-encore)? It's basically just a wrapper that makes configuring Webpack easier and integrates very well with Symfony apps. But all it does at the end of the day is stuff Webpack can do.

To start the dev server or run dev/prod builds, we use the Encore CLI (for example `encore dev` instead of `webpack build --mode=development`). At the end of the day the CLI is just [calling Webpack ](https://github.com/symfony/webpack-encore/blob/5cde9bef4a6075a1227c8635bea4580ec97b4480/bin/encore.js) behind the hood, but I don't know if there's an easy way to convert that to use Rspack instead.

**Comment by hardfist** [comment]
we have no experience about encore，so maybe you can try integrate rspack with encore and if you meet problems we can help

**Reply by YummYume** [reply]
Hi @hardfist thanks for the quick answer! Well, the thing is, I don't really know where to start or what I should try to do.

I was thinking that maybe https://modernjs.dev/builder/en could provide a solution for this use case, as in I would let Encore do its thing with the config and then call something like `await builder.build();` with the generated config. I'll try to experiment a little here.
